_includes/usage/cf-typography.md

```diff
@@ -45,7 +45,7 @@ project by duplicating the variable `@key: value`.
 
 ```
 @webfont-regular: 'AvenirNextLTW01-Regular';
-@webfont-italic: 'AvenirNextLTW01-Italic';
```

Comment:
I think it's worth adding a note to the docs (above or below here) about _why_ we're doing this, noting the fact that (a) faux italic Avenir Next Regular is almost indistinguishable from Avenir Next Italic, and (b) we don't use a lot of italic as a matter of course in our websites.

Reply:
Might roll it into https://github.com/cfpb/capital-framework/issues/700 if that makes more sense.

Reply:
Oy, just realized that this is the docs PR and it needs to be updated in the main branch's usage.md file first. Stand by.